docs/manual/execution-lifecycle/roi-metrics.md

```diff
@@ -0,0 +1,44 @@
+# ROI Metrics Data
+
+::: enterprise
+:::
+
+Automation provides value with each execution of a job.  A job can save time, dollars, and most likely a combination of both.  The ROI (return on investment) Metrics integration will help track the user-defined value of each job execution. **ROI Metrics Data** stores arbitrary key value pairs against your job to help understand your ROI per job execution. The metrics will be stored against the each execution of the job, and accessible via API for further analysis. There is also relevant default execution data to help you further understand your savings.
+
+## Configuring ROI Metrics
+
+The values stored are unique entries chosen by the user.  The values are abstracted from a specific metric type (hours, dollars, etc.) to allow customers to define their own ROI metrics.  Follow these steps when editing a job to configure ROI Metrics:
+
+1. Choose the _Execution Plugins_ tab.
+1. Check the box for **ROI Metrics Data**.<br><br>
+    ![ROI Metrics Plugin](~@assets/img/roi-metrics-data.png)
+1. Click **Add Custom Field**.<br><br>
+    ~[Add Custom Field](~@assets/img/roi-metrics-customfield.png)
+1. Enter a value for:
+    - ***Field Label***: This is the friendly name of the field that will be shown when picking a value
+    - ***Field Key***: This is the name of the field used in the programmatic output. _For simplified gathering later we suggest no spaces or special characters._
```

Comment:
Let's put the italicized text into a warning "note" so that it stands out more as done in the "Heads Up" section of [this page](https://docs.rundeck.com/docs/learning/solutions/automated-diagnostics/linux-diagnostics-example.html#example-linux-host-diagnostics)